tests/presenters/teaser-presenter.spec.js

```diff
@@ -12,6 +12,12 @@ describe('Teaser Presenter', () => {
 
 	context('classModifiers', () => {
 
```

Comment:
I'll add that test as a different test, because the condition we were trying to avoid isn't related to the mods that are external to the classModifiers — since we Object.assign(context, options.hash) in the nTeaserPresenter, the original mods get overridden. But we repeatedly push extra modifiers into the Presenter mods when we call classModifiers, which is where the reset comes in